src/evaluating_students/router.js

```diff
@@ -1,14 +1,363 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
+let GradeModel = require("./model/grade").GradeModel;
+let StudentModel = require("./model/student").StudentModel;
```

Comment:
Waarom is dit nog steeds let?

Reply:
Haha ja Github had iets van 11 van je comments verborgen met een "laad meer comments' knop. Die had ik gemist. Ben ze nu aan het fixen hahaha.